src/plugins.js

```diff
@@ -0,0 +1,117 @@
+'use strict'
+
+const { app, nativeImage } = require('electron')
+const { join } = require('path')
+const { warn, verbose, logger } = require('./common/log')
+const { uniq, pluck } = require('./common/util')
+const { promises: jsonld } = require('jsonld')
+
+
+class Plugins {
```

Comment:
`Plugins` is definitely a name I would pick as well; but I'm a bit concerned that it is too opaque so I usually end up changing plural names. Maybe something along the lines of `PluginManager`, which I don't particularly like. Or maybe even `TPM` (the `TropyPluginManager`)... what do you think?

Reply:
Agreed. The idea of `TPM` only made sense to me because it's similar to `NPM` (or `APM` for atom). Not keen on `PluginManager`, but it will probably age better than `Plugins`.